dace/transformation/subgraph/reduce_expansion.py

```diff
@@ -185,9 +185,9 @@ def expand(self, sdfg, graph, reduce_node):
 
         else:
             if self.debug:
-                print(f"ReduceExpansion::Expanding Reduction into Map"
-                      "and introducing update Tasklet,"
-                      "connecting with ancestor {array_closest_ancestor}")
+                print(f"ReduceExpansion::Expanding Reduction into Map "
+                      "and introducing update Tasklet, "
+                      "connecting with ancestor.")
```

Comment:
this will not work in Python (f-string without any expression to replace). Remove f